### ISSUE TITLE
Cant change the NavigationBar style

### ISSUE THREAD

**Issue by hazemmaher92** [freshdesk/hotline-ios#23]
`[[UINavigationBar appearance] setBarTintColor:[UIColor GreenColor]];
    [[UINavigationBar appearance] setTintColor:[UIColor whiteColor]];
    [[UINavigationBar appearance] setTitleTextAttributes:@{NSFontAttributeName: [UIFont systemFontOfSize:18],NSForegroundColorAttributeName: [UIColor whiteColor]}];`
i'am adding style to the UINavigationBar , but when I present `[[Hotline sharedInstance] showConversations:self];` or getting `[[Hotline sharedInstance] getConversationsControllerForEmbed]` and present it in a custom navigation bar it doesn't follow my style , and it creates a new style, that new style override my app style 

How to force that UINavigationBar to follow my style

**Comment by harish-kumarc** [comment]
Hi @hazemmaher92 
Hotline SDK contains a "HLTheme.plist" file under HLResource.bundle from where you can control the style(Color/Font/Images) of Hotline view into your app. It is quite straight forward, you just need to edit key values into your theme file. Please check [here](https://support.hotline.io/support/solutions/articles/221187-hotline-ios-theming-and-customization-guide) for more on customization.
For navigation bar changes please check keys in the attached screenshot here.
Thanks 

<img width="539" alt="screen shot 2017-10-02 at 2 18 12 am" src="https://user-images.githubusercontent.com/16202319/31058845-79adc100-a718-11e7-9f6d-b8788e576dff.png">

**Comment by hazemmaher92** [comment]
yes, doing that fixed the navigationBar Style , but after dismissing the Hotline ViewController (by tapping the close button) the default UINavigationBar title color changes to black in the whole app.

**Comment by harish-kumarc** [comment]
Hi @hazemmaher92 
Can you give us more information about your implementation like are using embed view api etc. Its good if you can share your Hotline implementation code snippet.
In NavigationBar theme we recommend to use same title color, font, size that you are using in your app so that it will match with style.
Thanks

**Comment by hazemmaher92** [comment]
UINavigationBar style in the AppDelegate
`
[[UINavigationBar appearance] setBarTintColor:[UIColor GreenColor]]; 
[[UINavigationBar appearance] setTintColor:[UIColor whiteColor]]; 
[[UINavigationBar appearance] setTitleTextAttributes:@{NSFontAttributeName: [UIFont systemFontOfSize:18],NSForegroundColorAttributeName: [UIColor whiteColor]}];
`
HotLine implementation
`[[Hotline sharedInstance] showConversations:self];`

Hotline configuration ( I have only changed the NavigationBar section matching my app color scheme)
![screen shot 2017-10-05 at 12 29 09 pm](https://user-images.githubusercontent.com/30255294/31220668-d63bc954-a9c9-11e7-9701-53f9b062be13.png)

My UINavigationBar before presenting the hotline
![img_2386](https://user-images.githubusercontent.com/30255294/31220709-f3225a1a-a9c9-11e7-9dc3-b73c3a5e969a.jpg)

My UINavigationBar after dismissing the hotline (by tapping the close button)
![img_2385](https://user-images.githubusercontent.com/30255294/31220731-04786a84-a9ca-11e7-893a-777dd6b77a0f.jpg)


thank a lot for your support

**Comment by harish-kumarc** [comment]
Hi @hazemmaher92 
I have a quick solution for the issue you are facing. Can you please change the navigation bar color in viewwillappear of the controller where you are presenting Hotline. Check code snippet here
```
-(void)viewWillAppear:(BOOL)animated{
    [super viewWillAppear:animated];
    [self.navigationController.navigationBar setTitleTextAttributes:
     @{NSForegroundColorAttributeName:[UIColor yellowColor]}];
}
```
We will surely analyse why default navigation title color is override.
